src/VAC/VectorAnimationComplex/VAC.cpp

```diff
@@ -2005,6 +2012,28 @@ void VAC::changeFacesColor()
         }
     }
 
+    if (emitCheckpoint)
+    {
+        checkPointTimer->stop();
+        checkPointTimer->start(5);
+    }
+}
+
+void VAC::changeInfillColor()
+{
+    auto emitCheckpoint = false;
+    auto infillColor = global()->infillColor();
+    for(auto cell : selectedCells())
+    {
+        if (auto keyFace = cell->toKeyFace())
+        {
+            emitCheckpoint = emitCheckpoint ? true : infillColor != keyFace->infillColor();
```

Comment:
emitCheckpoint || (infillColor != keyFace-> ... ) is a bit clearer to read, IMHO. :)